lib/akamai_rspec/request.rb

```diff
@@ -77,8 +77,11 @@ def get(url, headers = {})
         return AkamaiRSpec::Response.new(url)
       end
 
-      uri = parse_url_with_auth(url)
+      if url.class.ancestors.include? AkamaiRSpec::Response
```

Comment:
I'd like future developers to be able to `git blame` and find out WTF is going on here :/